### DISCUSSION TITLE
Projects view in iPad is not making use of the screen size

### DISCUSSION THREAD

**Discussion by IdanAdar** [community/community#136476] [Mobile]
iPad has a large screen but the mobile app is still showing it as if it's an iPhone screen, whereas it should display like it how it is on the web - a "table" with columns, rather than a single column with "separators"...

Cc @mattruggio

GitHub for iOS, Version 1.174.0 (139241358)
Device: iPad14,10 (iOS 17.6.1)

Current layout of a Project
![image](https://github.com/user-attachments/assets/9840af12-896c-4e7c-9078-f92f6fb94892)

**Comment by mattruggio** [comment]
Hey @IdanAdar, thanks for the mention! Projects support is a known parity mismatch between web and mobile experiences. There are currently open internal designs looking to improve the Projects's mobile UIX but, unfortunately, I cannot see/comment on their prioritization or timeline around such updates. As someone who uses Projects in my daily work, I would also love to see improvements to the mobile experience as well.
